packages/terra-detail-view/lib/DetailView.scss

```diff
@@ -0,0 +1,43 @@
+@import '~terra-mixins';
+@import './variables';
+
+.terra-DetailView {
+  padding: 1.428rem;
+
+  > * {
+    padding-bottom: 1.428rem;
+  }
+}
+
+.terra-DetailView-primaryText {
+  font-size: 1.428rem;
+  line-height: 1.428;
+  margin-bottom: 0.357rem;
+  margin-top: 0;
+}
+
+.terra-DetailView-footerText {
+  @include terra-text-align-center();
+  color: $terra-grey-50;
+  font-size: 1rem;
+  line-height: 1.428;
+}
+
+.terra-DetailView-divider {
+  border-color: $terra-grey-20;
+  border-style: solid;
+  border-width: 1px 0 0;
+  margin: 0;
+  padding-bottom: 0.714rem;
```

Comment:
At one point in time there was a discussion to limit rem values into specific intervals, I don't remember the number but something like factors of 0.15. I wouldn't worry to much about it right now, but there are a lot of crazy numbers floating around.